src/components/color_picker/color_picker.xml

```diff
@@ -41,38 +41,47 @@
             t-attf-style="background-color:{{color}};"
             t-on-click="() => this.onColorClick(color)">
             <div
-              t-if="props.currentColor === color"
+              t-if="isSameColor(props.currentColor, color)"
```

Comment:
:eyes: I thought this was already fixed

Reply:
same here, I recall the forwardport of the "+" sign was a bit messy, it might be a relicate